apps/electron/src/helper/dialog/dialog.ts

```diff
@@ -197,7 +201,22 @@ export async function loadDBFile(): Promise<LoadDBFileResult> {
       }
     }
 
+    if (validationResult === ValidationResult.MissingVersionColumn) {
```

Comment:
How do we read the version number from tables and migrate on demand? Or it is not planned for this pr? 🤔